pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -8,6 +8,11 @@ import (
 	"strconv"
 
 	"github.com/go-logr/logr"
+	litmuschaosv1alpha1 "github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
```

Comment:
This package is imported two times, other at https://github.com/litmuschaos/chaos-operator/pull/49/files#diff-1ec216d1fe3eec0d48ad9be24025c1a8R30